### ISSUE TITLE
"v bug" no longer works

### ISSUE THREAD

**Issue by elliotchance** [vlang/v#20244]
### Describe the bug

I tried to search if anyone else has raised the bug but the terms were too common. So please close this as a duplicate if that's the case.

### Reproduction Steps

```
v bug bug.v
```

Follow the prompts, and it will open up the Github page here: https://github.com/vlang/v/issues/new/choose and none of the fields will be filled in.

This has been broken for a while, I hope it's just the URL that needs to be updated.

### Expected Behavior

Open a bug template and fill in most fields.

### Current Behavior

Opens the wrong page.

### V version

V 0.4.3 ed754cf

### Environment details (OS name and version, etc.)

```
V full version: V 0.4.3 4bc9a8f.ed754cf
OS: macos, macOS, 12.6, 21G115
Processor: 10 cpus, 64bit, little endian, Apple M1 Max

getwd: /Users/elliot/Library/Mobile Documents/com~apple~CloudDocs/Development/github.com/elliotchance/vsql/tests
vexe: /Users/elliot/Downloads/v-5/v
vexe mtime: 2023-12-21 14:58:49

vroot: OK, value: /Users/elliot/Downloads/v-5
VMODULES: OK, value: /Users/elliot/.vmodules
VTMP: OK, value: /tmp/v_501

Git version: git version 2.37.0 (Apple Git-136)
Git vroot status: weekly.2023.44-420-ged754cfc
.git/config present: true

CC version: Apple clang version 14.0.0 (clang-1400.0.29.102)
thirdparty/tcc status: thirdparty-macos-amd64 46662e20
```
> [!NOTE]
> You can use the 👍 reaction to increase the issue's priority for developers.
>
> Please note that only the 👍 reaction to the issue itself counts as a vote.
> Other reactions and those to comments will not be taken into account.

**Comment by spytheman** [comment]
Very good find!

Thanks for reporting it.

It turns out that `v bug` continued to work for members of https://github.com/vlang , but stopped for anyone else, after we introduced issue templates in 914007f (note the `blank_issues_enabled: false`, as @MCausc78 discovered).

**Comment by spytheman** [comment]
What is insidious about it, is that it continued to work for me (I just used `v bug` to file https://github.com/vlang/v/issues/20246 ).

**Comment by spytheman** [comment]
Fixed by cedf631 (github will allow again the bug reports sent by `v bug`, even though they use a different formatting/template).
